src/meta/src/hummock/vacuum.rs

```diff
@@ -112,6 +113,15 @@ impl VacuumManager {
         let mut batch_idx = 0;
         let batch_size = 500usize;
         let mut sent_batch = Vec::with_capacity(objects_to_delete.len());
+        if self.env.opts.enable_hummock_data_archive {
+            if !objects_to_delete.is_empty() {
+                warn!(
+                    ?objects_to_delete,
```

Comment:
This log can be huge when `enable_hummock_data_archive = true`. We can log the object counts instead.